crates/net/network/src/discovery.rs

```diff
@@ -232,6 +230,17 @@ impl Discovery {
         }
     }
 
+    fn on_discv5_update(&mut self, update: Event) {
+        match update {
+            Event::Discovered(enr) => {}
+            Event::EnrAdded { enr, replaced } => {}
+            Event::NodeInserted { node_id, replaced } => {}
```

Comment:
here we need to handle the discovered entries

see https://github.com/paradigmxyz/reth/blob/32bf97d7bc7a862e5440f3182428de427585bb0f/crates/net/network/src/discovery.rs#L169-L169